### ISSUE TITLE
lane segment coordinates fluctuation in VectorMap

### ISSUE THREAD

**Issue by Shi-Qi-Li** [motional/nuplan-devkit#107]
Hi, I save some VectorMap objects and plot the maps based on the data in VectorMap and I found that some lanes have fluctuated coordinates.
![lALPJx8ZxmV5uyDNBTjNBWg_1384_1336 png_720x720q90g](https://user-images.githubusercontent.com/29630040/185833005-fe89daa7-6643-4850-bb2c-f009fa624091.jpg)
I wonder if this fluctuation is normal.

Here is my code to plot the map.
```py
import matplotlib.pyplot as plt
for group in lane_groupings:
    for id in group:
        plt.plot([coords[id][0][0], coords[id][1][0]], [coords[id][0][1], coords[id][1][1]], color='black')
plt.show()
```

**Comment by patk-motional** [comment]
Hi @Shi-Qi-Li,

This is very strange. Can you show how you created the VectorMap feature? We will try to re-create you issue

**Comment by Shi-Qi-Li** [comment]
> Hi @Shi-Qi-Li,
> 
> This is very strange. Can you show how you created the VectorMap feature? We will try to re-create you issue

I add these codes to `_step()` in `nuplan/planning/training/modeling/lightning_module_wrapper.py`
```py
import numpy as np
vector_map_data = features["vector_map"]
ego_agents_feature = features["agents"]
target_trajectorys = targets['trajectory']

vector_map = {
    "coords":vector_map_data.coords,
    "lane_groupings":vector_map_data.lane_groupings,
    "multi_scale_connections":vector_map_data.multi_scale_connections,
    "on_route_status":vector_map_data.on_route_status,
    "traffic_light_data":vector_map_data.traffic_light_data
}

agents = {
    "ego": ego_agents_feature.ego,
    "agents": ego_agents_feature.agents
}

target_trajectory = {
    "xy": target_trajectorys.xy,
    "heading": target_trajectorys.heading
}

np.savez_compressed(f'/xxx/vector_map/vector_map.npz',**vector_map)
np.savez_compressed(f'/xxx/agents/agents.npz',**agents)
np.savez_compressed(f'/xxx/target_trajectory/target_trajectory.npz',**target_trajectory)
```

**Comment by patk-motional** [comment]
Can you share with us the log and token of where you encountered this? We're still trying to recreate your issue

**Comment by Shi-Qi-Li** [comment]
> Can you share with us the log and token of where you encountered this? We're still trying to recreate your issue

Sorry I can't find the token corresponding to the upper picture, but there are some similar maps, such as the `42c151ff471d5209`, `a2291c844b785ef8` in `2021.08.30.14.54.34_veh-40_00439_00835` and `cd982e0d1f125986`, `cd982e0d1f125986`, `461e661e7d3f598b` in `2021.05.12.22.00.38_veh-35_01008_01518`. 
You can load the `.gz` file in `\exp\cache\log\token\` to recreate my issue.

**Comment by patk-motional** [comment]
Hi, 
We have tried the same thing and we have not been able to recreate your issue. Can it have to do something with compressing the feature as you have in the example above?

**Comment by Shi-Qi-Li** [comment]
> Hi, We have tried the same thing and we have not been able to recreate your issue. Can it have to do something with compressing the feature as you have in the example above?

Hi, I solved the problem by updating to `v0.5`, it seems to be a bug in `v0.3`.
Thanks for your help.
